apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx

```diff
@@ -0,0 +1,597 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { StoryWright, Steps } from 'storywright';
+import { TestWrapperDecorator } from '../../utilities/TestWrapperDecorator';
+import { AreaChart, ICustomizedCalloutData, ChartHoverCard } from '@fluentui/react-charting';
+import { DefaultPalette } from '@fluentui/react';
```

Comment:
Can we add 1 more screenshot for following:
Hover over basic area chart, hover over multiple area chart.